<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_interviewer</class>
 <widget class="QDialog" name="dialog_interviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>面试官注册</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>231</width>
       <height>391</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_line"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>注册密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_line"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>再次确认密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_again"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>校验码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="key"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="register_button">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>490</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注册面试官</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>面试官登录</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>181</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>面试官用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_login"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>登录密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_login"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="interviewer_login">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>400</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>面试官登录</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
